src/plugins/resolver/resolver.c

```diff
@@ -549,6 +550,13 @@ int ELEKTRA_PLUGIN_FUNCTION (get) (Plugin * handle, KeySet * returned, Key * par
 	/* Start file IO with stat() */
 	if (stat (pk->filename, &buf) == -1)
 	{
+		if (pk->hasExisted)
+		{
+			pk->hasExisted = 0;
+			// was not missing before
+			return /* new return value required */
```

Comment:
I'm guessing you want to communicate the state to kdbGet using a return value?

The expected return values (correct me if I'm wrong) for this function would be these here 

https://github.com/ElektraInitiative/libelektra/blob/71a10abf0baa83dff82b44f5769159c840612e9b/src/include/kdbplugin.h#L83-L92

So just introducing a new Macro, returning it and then handling the state in `libelektra-kdb` is fine

Reply:
Correct, but as none of the existing return values cover the case of a removed file, we have to introduce a new return value.